.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: Run tests
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    name: ${{ matrix.name}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with minimal dependencies
+            toxenv: py39-test
+          - os: ubuntu-latest
+            python-version: 3.9
+            name: Python 3.9 with no visualization + coverage
+            toxenv: py39-test-novis-cov
```

Comment:
`-novis` isn't defined in the tox file.

https://github.com/radio-astro-tools/pvextractor/blob/master/tox.ini#L3

py39 also needs to be added there.